components/modals/AddOptionModal.tsx

```
@@ -0,0 +1,197 @@
+                                    Titre
+                                </Text>
+                                <TextInput
+                                    className="border border-gray-300 dark:border-gray-600 rounded-lg p-3 bg-white dark:bg-gray-700 text-black dark:text-white"
+                                    placeholder="Titre du logement"
+                                    value={newOption.title as string || ""}
+                                    onChangeText={(t) => setNewOption({ ...newOption, title: t })}
+                                />
+                            </View>
+                            <View>
+                                <Text className="text-sm font-medium mb-1 dark:text-gray-300">
+                                    URL de l'image
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)